src/Data/Number.purs

```diff
@@ -53,3 +84,114 @@ fromString :: String -> Maybe Number
 fromString str = runFn4 fromStringImpl str isFinite Just Nothing
 
 foreign import fromStringImpl :: Fn4 String (Number -> Boolean) (forall a. a -> Maybe a) (forall a. Maybe a) (Maybe Number)
+
+-- | An alias to make types in this module more explicit.
+type Radians = Number
```

Comment:
I am anti type synonyms like this; they suggest type safety where none exists, and in the end only really require you to keep more information in your head (namely, that `Radians` is just an alias for `Number`). I think now is a good opportunity to get rid of this type synonym.

Reply:
I agree. This was copied over from purescript-math but now is a good time to make improvements. Fixed in 119cf59c24dbb4c5ecaf5610cf753f36f918d83e